types/jest/index.d.ts

```diff
@@ -21,6 +21,8 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
+import jestDiff = require('jest-diff');
```

Comment:
do I need to declare this dependency anywhere?

Reply:
I've not introduced a dependency before either, but looking at this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest-when/index.d.ts I think you need that triple-slash reference

Reply:
The main issue with all the tests blowing up is that this file is now treated as a module by the type checker now that it has a top-level `import`, so its types are no longer global. Thus the thousands of "cannot find 'expect'" errors. You should be able to use a local/dynamic import type. I can't remember the exact syntax, or even the right name for it. But something like:
```ts
diff: typeof import('jest-diff');
```

The triple-slash may or may not be needed. I know some types have package.jsons with dependencies specified.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped#im-writing-a-definition-that-depends-on-another-definition-should-i-use-reference-types--or-an-import

I'll try the `typeof import` 🙂

Reply:
oh, adding `dependencies: {"@types/jest-diff"}` throws an error saying package shouldn't depend on stuff from `types`, or something like that, I forget